packages/yoroi-extension/app/styles/themes/modern-theme.js

```diff
@@ -46,6 +46,7 @@ const theme = {
     MuiChip: ModernChip,
     MuiTooltip: ModernTooltip,
   },
+  spacing: [0, 2, 3, 5, 8],
```

Comment:
This file is not mantained anymore theoretically, is for the old modern theme.

Reply:
yeah, will remove this